client/src/components/SearchForm.js

```diff
@@ -1,11 +1,44 @@
 import React, { Component } from 'react';
+import Select from 'react-select';
+import 'react-select/dist/react-select.css';
+const data = require('../citydata');
```

Comment:
change this from a `require` to `import`

Reply:
also use destruction-ring to `{ data  }` so you won't have to do `data.data` [here](https://github.com/FACN3/Israel_Housing/pull/28/files#diff-75814a47c242c9dc4c66e9f653233c55R36)

Reply:
Yeah. That was a good opportunity to use es6.

Reply:
ok ? where is it ?